.github/workflows/CI.yml

```diff
@@ -215,7 +215,11 @@ jobs:
       - python-lint
       - python-tests
       - fetch-migration-docker-build
+    if: always()
     runs-on: ubuntu-latest
     steps:
+      - if: ${{ contains(needs.*.result, 'cancelled') || contains(needs.*.result, 'failure') || contains(needs.*.result, 'skipped') }}
```

Comment:
When is a job "skipped"?

Say, for example, we have our CDK tests run only if a file in the CDK directory was changed, would that register as skipped for PRs that don't include that?

Reply:
We don't have any jobs that are skipped.  If we did want to support conditional execution of some of our jobs we would need to change this code.  Note; I couldn't check for `every(needs.*.result, 'success')` which would have been preferred

Reply:
This clearly works with the way our current code is setup. What I was trying to get at would be if excluding `|| contains(needs.*.result, 'skipped')` would be more defensive assuming users can't intentionally skip tasks or error cases aren't coded as skipped